app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -254,11 +308,12 @@ export default {
       confirmSubmitModal: false,
       formSubmitLoading: false,
       formSubmitSuccess: false,
+      formSubmitError: false,
       saveFormSuccess: false,
       loadFormSuccess: false,
       confirmLoadModal: false,
       step: 1,
-      maxSteps: 11, // total number of wizard steps
+      maxSteps: 15, // total number of wizard steps
```

Comment:
We need to think of a nice way to calculate this automatically based on the number of steps that have been added in as components - the solution we go for here is probably associated with the auto-numbering solution.